packages/component/src/SendBox/MicrophoneButton.js

```diff
@@ -12,7 +12,17 @@ import MicrophoneIcon from './Assets/MicrophoneIcon';
 const { DictateState } = Constants;
 
 const ROOT_CSS = css({
-  display: 'flex'
+  display: 'flex',
```

Comment:
Can you check if we need `display: flex` here?

Reply:
Looks like it's needed in Edge, if it's not there a scrollbar appears on Web Chat (for the whole window)
Leaving as is!